packages/service-integration/src/service_integration/pytest_plugin/docker_integration.py

```diff
@@ -0,0 +1,301 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+import filecmp
+import json
+import os
+import shutil
+import urllib.request
+from pathlib import Path
+from pprint import pformat
+from typing import Dict
+
+import docker
+import jsonschema
+import pytest
+import yaml
+
+_FOLDER_NAMES = ["input", "output"]
+_CONTAINER_FOLDER = Path("/home/scu/data")
```

Comment:
Not all images have this home directory. Why impose it? Maybe I am mistaken but I do not understand what this var refers to.

Reply:
I took this verbatim from the current setup. In this first iteration I want to be 100% compatible with the cookie-cutter
I will add a note there to check though. thx